flax/linen/module.py

```diff
@@ -1214,6 +1214,8 @@ def other_fn(instance, ...):
       *args: Named arguments passed to the specified apply method.
       rngs: a dict of PRNGKeys to initialize the PRNG sequences.
         The "params" PRNG sequence is used to initialize parameters.
+        For example :class:`flax.linen.stochastic.Dropout` requires an
```

Comment:
Wondering if we should reference a guide instead of just dropout, currently there is no specific guide however.